src/swig/SwigInterface.i

```diff
@@ -0,0 +1,64 @@
+ %module SwigInterface
+ %include "std_string.i"
+ %include "std_map.i"
+ %include <std_vector.i>
+
+namespace std {
+    %template(map_string_string) map<string, string>;
+}
+
+/**
+ * Enables the reading and writing of the input and output inside a Python program
```

Comment:
Does this work for Java or just Python? If just Python, remove it. All interfaces should be the same, special functionality should not exist on a per-language basis.